.github/workflows/ci.yaml

```diff
@@ -0,0 +1,51 @@
+name: Continuous Integration
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  linux-mono:
+    name: Linux (mono)
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v2
+
+      - name: Prepare Environment
+        run: |
+          . mod.config;
+          awk '/\r$$/ { exit(1); }' mod.config || (printf "Invalid mod.config format. File must be saved using unix-style (LF, not CRLF or CR) line endings.\n"; exit 1);
+
+      - name: Check Code
+        run: |
+          make check
+          make check-packaging-scripts
+
+      - name: Check Mod
+        run: |
+          sudo apt-get install lua5.1
+          make check-scripts
+          make test
+
+  windows:
+    name: Windows (Framework 4.7)
+    runs-on: windows-2019
+
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v2
+
+      - name: Check Code
+        shell: powershell
+        run: |
+          dotnet nuget locals all --clear
+          .\make.ps1 check
```

Comment:
This alone will lead to
> OpenRA.Utility.exe could not be found. Build the project first using the `"all`" command.

Reply:
looks like this is a long-standing bug in `make.ps1`. the Check command doesn't chain through to the engine...